torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2359,7 +2359,7 @@ def _maybe_evaluate_static_worker(
         # we have to increase it by offset (and conversely, the new
         # variables have to have their value range bounds adjusted as
         # well)
-        s = sympy.Symbol(f"evaluate_static_shape_{idx}", positive=True, integer=True)
+        s = sympy.Symbol(f"guard_int_{idx}", positive=True, integer=True)
```

Comment:
where is the guard happening? it seems off to me that you are saying we are guarding, which is a non pure function, in a pure function?

Reply:
this change is wrong. a log message happened to match the function name